sass/components/_chart-controls.scss

```diff
@@ -212,3 +229,57 @@
         }
     }
 }
+.smartcharts-480 {
```

Comment:
place these classes instead of media-query, so that chart control change by the size of chart container but not the screen.

Reply:
this looks good to me :+1: